modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -341,7 +352,9 @@ private void addNpmPackageGeneration(SemVer ngVersion) {
         }
 
         // set zone.js version
-        if (ngVersion.atLeast("15.0.0")) {
+        if (ngVersion.atLeast("16.0.0")) {
+            additionalProperties.put("zonejsVersion", ">=0.11.5,<=0.14");
```

Comment:
not sure how exactly it's used in the templates. but the min required version is 0.13.x, as per the migration guide:
https://update.angular.io/?l=3&v=15.0-16.0
> Angular v16 supports Zone.js version 0.13.x or later.

Reply:
Indeed, I've fixed the dependencies specifications.